test/evergreen.yml

```diff
@@ -4649,6 +4922,46 @@ tasks:
         vars:
           test_path: bench/wt2853_perf/wt2853_perf
 
+  - name: aggregate-perf-stat-long
```

Comment:
Can we merge all of these aggregate tasks into a single job? That way we have a single .csv file to look at in the end.